### DISCUSSION TITLE
看下巴就行了？辨識無聲語言的項鍊來了

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#528] [科技新知]
想像一下，如果你坐在掉針可聞的會議室或圖書館裡，脫口而出「Siri，明天天氣如何」，或許不太合適。當一個人不能說話或必須保持安靜，卻想對智慧設備下指令，該怎麼辦？_
康乃爾大學訊息科學助理教授_ Cheng Zhang 和博士生 Ruidong Zhang 設計了叫「SpeeChin」的項鍊。它基於下巴運動，捕捉「頸部和臉部皮膚變形圖」，辨識無聲語言，目前可辨識英語和中文簡單短詞。
未來我想這個技術也許可以應用於心理健康領域，偵測一天的心情變化，方便心理醫師治療。_

https://technews.tw/2022/02/27/smart-necklace-recognizes-english-mandarin-commands/

**Comment by xkllkx** [comment]
話說，好像有抗躁口罩這個東西。

**Comment by sean1325418** [comment]
原來看下巴也能辨識在說甚麼，還以為只能靠唇語才能辨識

**Comment by etin2468** [comment]
用下巴判斷感覺能得到的判斷資訊很有限ㄟ，到底怎麼做到的?

**Comment by Yuhsin000** [comment]
然後siri就在安靜的環境大聲回覆使用者 : 我不知道你在說甚麼。對我而言，我不太能放心使用這個功能

**Comment by ghost** [comment]
這感覺有點不太必要，有更多能取代並且發展性更好的東西了@@

**Comment by ThomasMeng0515** [comment]
文中有提到當受測者在移動時，成功辨別的機率就比較低，所以似乎只能用在受試者靜止的情況下，這樣好像有點不太實用

**Comment by chia-kai-chang** [comment]
我喜歡這個專題創作，而且可以用創客方式實作，希望之後的AI專題學生有機會做出類似的AI專題
要怎麼做優文收藏呢~ 我來研究一下~

**Reply by chia-kai-chang** [reply]
架一個討論區來記錄優文好了 @AngusGuei

**Comment by lx3211** [comment]
這想法其實很好但感覺屬於非必需品
如果有捲舌音的部分
透過下巴也可以辨別嗎

**Comment by cordydeng** [comment]
這個功能感覺很炫 , 不過不知道實用性高不高 , 將來如果這個技術更加成熟 , 搞不好會是個好技術

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望未來可以廣泛使用。

**Comment by henrywen547** [comment]
這功能感覺很厲害!
不過還是要以精確為主啦
而且辨識嘴型應該比較準，用途也比較多
但疫情之下不能脫口罩，這個功能若能發展完善
確實也能幫助不少人!

**Comment by HsuChen88** [comment]
有個問題，不開口的情況下嘴巴所有部位的活動度很低，好像沒辦法講原本的語言呢，意思是沒辦法做出那些動作，那他判斷的也不是原本說話的下巴動作了，難道還是其實它是要我們學另一種專門的下巴語？！

**Comment by hapiwangy** [comment]
太有趣了，原來從下巴的形變也可以發出指令，'

**Comment by bloodtrail0511** [comment]
酷欸，如果辨識技術持續完善，這樣既可以保證準確性，也可以保障隱私

**Comment by yucheng1028** [comment]
太有趣了，這樣霍金多一種說話方式了

**Comment by sys511613** [comment]
上課時可以無聲聊天的日子指日可待！

**Comment by minniewei** [comment]
喔喔喔！！期待我未來能我真的運用這個東西

**Comment by treeann** [comment]
不得不說這個發明真的厲害

**Comment by stan890425** [comment]
那如果有雙下巴可以一次操縱兩個嗎?

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
這樣蠻方便的，尤其是像我這樣怕尷尬的人來說就很好用。

**Comment by Ben6uo** [comment]
感覺很酷 但會不會有判別錯誤的問題?

**Comment by Lynn12378** [comment]
感覺錯誤率夠低的話會很有幫助，畢竟有很多場合其實不適合發出聲音，或是雜音太多很難辨識，這個科技可以很好的解決問題。

**Comment by Jimmy911218** [comment]
感覺能避掉一些尷尬但不一定準確的感覺

**Comment by 110409538** [comment]
很酷的技術,多人聚會想要開小差的時候或許可以派上用場.
